editor/property_selector.cpp

```diff
@@ -210,6 +210,9 @@ void PropertySelector::_update_search() {
 
 				methods.push_back(MethodInfo("*Script Methods"));
 				Object::cast_to<Script>(obj)->get_script_method_list(&methods);
+				// TODO: Filter out things unsuitable for explicit calls, like constructors.
+
+				// TODO: We should handle script class hierarchies somehow. Right now we don't; it's unclear where this functionality should live.
```

Comment:
As far as I know, `get_script_method_list` should only return the methods declared in the scripts, not inherited methods. So this shouldn't be needed.

Reply:
In theory it should be returning methods declared in the script, and methods declared in parents of the script, up to (but not including) the first native class. Right now it only returns methods in the script itself. But I've removed the comment.